filters/fadein/fadein.go

```diff
@@ -192,14 +197,20 @@ func (endpointCreated) CreateFilter(args []interface{}) (filters.Filter, error)
 func (endpointCreated) Request(filters.FilterContext)  {}
 func (endpointCreated) Response(filters.FilterContext) {}
 
-// NewPostProcessor creates post-processor for maintaining the detection time of LB endpoints with fade-in
-// behavior.
-func NewPostProcessor() routing.PostProcessor {
+// NewPostProcessorWithOptions creates post-processor for maintaining
+// the fade-in behavior.
+func NewPostProcessorWithOptions(o Options) routing.PostProcessor {
 	return &postProcessor{
-		detected: make(map[string]detectedFadeIn),
+		detected:         make(map[string]detectedFadeIn),
+		endpointRegistry: o.EndpointRegistry,
 	}
 }
 
+// Deprecated: use NewPostProcessorWithOptions
```

Comment:
please use proper godoc (start with func name): 
```
NewPostProcessor *deprecated*: use NewPostProcessorWithOptions
```

Reply:
I think it's better not to use godoc starting with func name for deprecation to really work.
```
rzavodskikh@ZALANDO-39125:~/skipper$ staticcheck -checks "all,-ST1000,-ST1003,-ST1012,-ST1020,-ST1021" ./... # Start with "Deprecated"
skipper.go:1903:4: fadein.NewPostProcessor is deprecated: use NewPostProcessorWithOptions  (SA1019)
rzavodskikh@ZALANDO-39125:~/skipper$ staticcheck -checks "all,-ST1000,-ST1003,-ST1012,-ST1020,-ST1021" ./... # Start with "NewPostProcessor"
rzavodskikh@ZALANDO-39125:~/skipper$ 
```